ios/RNGoogleMobileAds/RNGoogleMobileAdsAppOpenModule.swift

```diff
@@ -74,6 +74,7 @@ class RNGoogleMobileAdsAppOpenModule: NSObject {
       GADAppOpenAd.load(
         withAdUnitID: adUnitId,
         request: adRequest,
+        orientation: UIApplication.shared.statusBarOrientation,        
```

Comment:
This is deprecated since SDK 9.14.0.
See release notes: [9.14.0](https://developers.google.com/admob/ios/rel-notes)